apps/auth/src/app/components/sign-x-via-google/sign-x-via-google.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TuiButtonModule } from '@taiga-ui/core';
+
+@Component({
+  selector: 'oak-sign-x-via-google',
+  standalone: true,
+  imports: [
+    CommonModule,
+    TuiButtonModule
+  ],
+  templateUrl: './sign-x-via-google.component.html',
+  styleUrls: ['./sign-x-via-google.component.scss']
+})
+export class SignXViaGoogleComponent implements OnInit {
+  @Input() x = 'In' 
```

Comment:
why x?, give it a more descriptive name like `action`. don't be using x

and the name of the component should be `GoogleAuthButton`